src/ocean/util/serialize/contiguous/Serializer.d

```diff
@@ -622,6 +622,7 @@ struct Serializer
                 {
                     // element is a dynamic array
                     data = This.dumpArray(element, data);
+                    *(cast(T*)&element) = null;
```

Comment:
This is going to be a problem with https://github.com/sociomantic-tsunami/ocean/pull/88 though - or do we have a way to reset all pointers to null in-place?